src/devtools/client/webconsole/utils/connected-object-inspector.tsx

```diff
@@ -5,23 +5,29 @@ import { SmartTraceStackFrame } from "devtools/client/shared/components/SmartTra
 import { actions } from "ui/actions";
 import { ObjectInspector, ValueItem, REPS, MODE } from "devtools/packages/devtools-reps";
 import SmartTrace from "devtools/client/webconsole/utils/connected-smart-trace";
+import { UIState } from "ui/state";
+import { getCurrentPoint } from "ui/reducers/app";
 
-type ObjectInspectorProps = PropsFromRedux & {
+interface PropsFromParent {
   value: ValueFront;
   useQuotes?: boolean;
   transformEmptyString?: boolean;
   escapeWhitespace?: boolean;
   style?: React.CSSProperties;
-};
+}
+type ObjectInspectorProps = PropsFromRedux & PropsFromParent;
```

Comment:
Could that be what we want, to basically invalidate an entire thing instead of just the items?
I understand the reasoning, and I'm fine with both ways (reactive vs passive) as both seem reasonable on the UX side as well.